packages/docs/operations-manual/key-management/detailed.md

```diff
@@ -0,0 +1,148 @@
+celocli account:show $ADDRESS_TO_DESIGNATE
+```
+
+## Authorized Vote Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a vote signing key. Authorized vote signing keys can vote in validator elections and on-chain governance on behalf of the Locked Gold Account.
+
+Note that the vote signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
+
+Authorized vote signing keys will only be used to send transactions and **can be stored on a Ledger hardware wallet**.
+
+
+### Authorizing a Vote Signer
+An undesignated key may be authorized as a vote signer on behalf of a Locked Gold Account by running the following commands:
+
+```shell
+# Create a proof-of-possession. Note that the signer private key must be available.
+celocli account:proof-of-possession --account $LOCKED_GOLD_ACCOUNT --signer $KEY_TO_AUTHORIZE --useLedger
+
+# Authorize the vote signing key. Note that the Locked Gold Account private key must be available.
+celocli account:authorize --from $LOCKED_GOLD_ACCOUNT --role vote --signer $KEY_TO_AUTHORIZE --signature $PROOF_OF_POSSESSION --useLedger
+
+# Confirm that the vote signer was authorized
+celocli account:show $LOCKED_GOLD_ACCOUNT
+
+# You can also look up account info via the authorized signer
+celocli account:show $KEY_TO_AUTHORIZE
+```
+
+## Authorized Validator Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a validator signer. Authorized validator signing keys can be used to register and manage a validator or validator group on behalf of the Locked Gold Account. If the authorized validator key is used to register a validator it is also used to sign consensus messages.
+
+### Authorized Validator Signers for Validator Groups
+An authorized validator signer key that will be used to register a validator group can be used to send group management transactions (e.g. register, add member A, queue commission update to 0.25, etc.) Because this key does not participate directly in consensus it **can be stored on a Ledger hardware wallet.**
+
+### Authorized Validator Signers for Validators
+An authorized validator signer key that will be used to register a validator can be used to send validator management transactions (e.g. register, affiliate with group A, etc.) This key will also be used to sign consensus messages and thus *cannot be stored on a Ledger hardware wallet* as signing consensus messages is not currently supported by the Celo Ledger App.
+
+Note that the validator signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
+
+
+### Authorizing a Validator Signer
+An undesignated key may be authorized as a validator signer on behalf of a Locked Gold Account by running the following commands:
```

Comment:
note should we use `LockedGold` to refer to the contract and add a link somewhere? It is referenced a lot

Reply:
Linked to the docs page on locked gold